playground-env/src/main/scala/org/ergoplatform/playgroundenv/utils/ErgoScriptCompiler.scala

```diff
@@ -24,4 +26,36 @@ object ErgoScriptCompiler {
     val prop = compiler.compile(liftedEnv, ergoScript)
     ErgoContract(_ => ???, prop)
   }
+
+  def printErgoScriptEncodingsAndHashesFromUrl(url: String) = {
+    val script           = scala.io.Source.fromURL(url).mkString
+    val ergoTree         = compile(Map(), script).ergoTree
+    implicit var encoder = new ErgoAddressEncoder(MainnetNetworkPrefix)
+    var addr             = Pay2SAddress(ergoTree)
+
+    val addrTypePrefix = Pay2SAddress.addressTypePrefix
+    val mainnetEncoded = encoder.toString(addr)
+    val mainnetHash = Base58.encode(
```

Comment:
I don't think we need network prefix and addrTypePrefix. I was thinking of just a hash of the serialized ErgoTree.